ra_aid/__main__.py

```
@@ -1,118 +1,194 @@
+from ra_aid.tools.memory import _global_memory
 
 logger = get_logger(__name__)
 
+
+def launch_webui(host: str, port: int):
+    """Launch the RA.Aid web interface."""
+    from ra_aid.webui import run_server
+    print(f"Starting RA.Aid web interface on http://{host}:{port}")
+    run_server(host=host, port=port)
+
 def parse_arguments(args=None):
```

Ruff (0.8.2)

`parse_arguments` is too complex (11 > 10)

(C901)